pkg/new-ui/v1beta1/backend.go

```diff
@@ -37,8 +40,20 @@ func NewKatibUIHandler(dbManagerAddr string) *KatibUIHandler {
 		log.Printf("NewClient for Katib failed: %v", err)
 		panic(err)
 	}
+	// create a new client for manipulating SAR objects.
+	conf, err := config.GetConfig()
+	if err != nil {
+		log.Printf("Failed to create k8s rest config: %v", err)
+		panic(err)
+	}
+	sarclient, err := kubernetes.NewForConfig(conf)
+	if err != nil {
+		log.Printf("SarClient for Katib failes: %v", err)
+		panic(err)
+	}
```

Comment:
Do you need to create new K8s client here ?
You can get controller runtime client from the Katib client using [`GetClient`](https://github.com/kubeflow/katib/blob/0377ed3932502e658e22a1a553f82db19280bab5/pkg/util/v1beta1/katibclient/katib_client.go#L36) API. You can use this client to create SAR object.

Reply:
Fixed!